### PR TITLE
improvement(cluster): create ccm cluster

### PR DESCRIPTION
**Pull request by juliayakovlev**
Create and start CCM cluster
 
Task: https://github.com/scylladb/qa-tasks/issues/1136

Test:
https://jenkins.scylladb.com/job/scylla-staging/job/yulia/job/staging-rust-driver-matrix/153/

Reported in Argus:
https://argus.scylladb.com/test/bd78a051-9bdd-4f70-b554-b881cd3cf1b7/runs?additionalRuns[]=c50639c2-08d2-493f-84b0-b86b766589bb

Screenshot from 2024-04-25 16-08-25

### CODE REVIEW

**Comment by fruch** [comment]
@juliayakovlev this one commit, that does much more than what's written on the commit message

and you split it into the relevant commits ?, i.e. changing docker, changing the the build command, and so on ?

File: common.py [hunk 1]
```diff
@@ -11,14 +11,17 @@
...
         status_code = proc.returncode
     assert status_code == 0, stderr
 
 
 def get_nodes_ip_from_ccm():
-    lines = subprocess.check_output('ccm status -v | egrep "node|binary"', shell=True).decode().splitlines()
+    try:
+        lines = subprocess.check_output('ccm status -v | egrep "node|binary"', shell=True, stderr=subprocess.STDOUT).decode().splitlines()
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError("command '{}' return with error (code {}): {}".format(e.cmd, e.returncode, e.output))
```
Comment (soyacz) [inline common.py:24]: `TestCluster` object already has property `ip_addresses` why not to use that?

File: main.py [hunk 2]
```diff
@@ -93,27 +89,27 @@ def extract_n_latest_repo_tags(repo_directory: str, major_versions: List[str], l
...
     parser.add_argument('--scylla-version', help="relocatable scylla version to use",
                         default=os.environ.get('SCYLLA_VERSION', None)),
-    parser.add_argument('--version-size', help='The number of the latest versions that will test.'
-                                               'The version is filtered by the major and minor values.'
-                                               'For example, the user selects the 2 latest versions for version 4.'
-                                               'The values to be returned are: 4.9.0-scylla-1 and 4.8.0-scylla-0',
+    parser.add_argument('--rust-driver-versions-size', help='The number of the latest versions that will test.'
+                                                            'The version is filtered by the major and minor values.'
+                                                            'For example, the user selects the 2 latest versions for version 4.'
+                                                            'The values to be returned are: 4.9.0-scylla-1 and 4.8.0-scylla-0',
```
Comment (soyacz) [inline main.py:112]: comment is refferring to different driver - rust is `0.12...` or soon `1.0...` how 1.0 will be catched?

Comment (juliayakovlev) [reply main.py]: comment fixed

Comment (juliayakovlev) [reply main.py]: It defined in `main.extract_n_latest_repo_tags`: first it is sorted by major version ("0", "1"..). Looks like shouldn't be a problem

File: common.py (hunk 1 above)
Comment (juliayakovlev) [reply common.py]: `get_nodes_ip_from_ccm` is not in use. Removed it

**Comment by juliayakovlev** [comment]
Test ran and passed (1 test failed, but it is not the matrix problem, I'll open issue in the driver repo).

Email sent:
Screenshot from 2024-04-25 16-16-03

Reported in Argus:
Screenshot from 2024-04-25 16-08-25

**Pull request merged**